### DISCUSSION TITLE
Suggestion - Fix Skill Calculator Sepulchre Values

### DISCUSSION THREAD

**Discussion by liebzer** [runelite/runelite#15974] [Q&A]
Currently the XP gained for hallowed sepulchre is not cumulative to include the previous floors in the calculator.  For instance the XP gained for floor 2 shows only the XP gained for floor 2 not floor 2 and 1 combined.  This does not make sense for this calculator since you cannot choose to do floor 2 and not floor 1.  Instead of the current values in the skill calculator I propose they change from the xp of each floor to the cumulative xp as presented on the wiki (https://oldschool.runescape.wiki/w/Hallowed_Sepulchre#Experience_rates).  This would give better estimates on number of laps/level.

**Comment by Enriath** [comment]
You can shift click to select multiple options already.
